### PR TITLE
Add CalculationSummary Titles to translations

### PR DESCRIPTION
**Pull request by dwyeradam**
Titles should be translatable for CalculationSummary blocks.
These are the main titles of the block but also the titles with the calculation object.

### CODE REVIEW

File: tests/schemas/test_translation.json [hunk 1]
```diff
@@ -1,206 +1,266 @@
...
+    "visible": true
+  },
+  "view_submitted_response": {
+    "enabled": true,
+    "duration": 900
+  },
+  "metadata": {
+    "user_id": {
+      "validator": "string"
     },
```
Comment (MebinAbraham) [inline tests/schemas/test_translation.json]: We missed the metadata changes for these -_-

**Comment by dwyeradam** [comment]
Possible I've only done half a job here. Need to check the translation script works.

File: tests/schemas/test_translation.json (hunk 1 above)
Comment (MebinAbraham) [reply tests/schemas/test_translation.json]: I've edited this in my PR anyway.

File: app/extract_translation_data.py [hunk 2]
```diff
@@ -79,6 +79,9 @@ def get_non_question_translatable_text(container):
         extracted_text.extend(get_introduction_translatable_text(container))
     elif container['type'] == 'Interstitial':
         extracted_text.extend(get_interstitial_translatable_text(container))
+    elif container['type'] == 'CalculatedSummary':
+        extracted_text.extend(get_titles_text(container['titles'], container['id']))
+        extracted_text.extend(get_titles_text(container['calculation']['titles'], container['id']))
```
Comment (dcdarrell9) [inline app/extract_translation_data.py]: Could possibly change this to `get_calculated_summary_text` or something like that.

But works fine and probably not worth changing 👍

**Pull request merged**